test/e2e/vic-helpers.js

```diff
@@ -0,0 +1,61 @@
+const mock = require('./mock-helpers');
+
+function completeApplicantInformation(client, data) {
+  client
+    .fillName('root_veteranFullName', data.veteranFullName)
+    .selectRadio('root_gender', data.gender)
+    .fill('input[name="root_veteranSocialSecurityNumber"]', data.veteranSocialSecurityNumber)
+    .fillDate('root_veteranDateOfBirth', data.veteranDateOfBirth)
```

Comment:
Unrelated to these changes, it looks like we're not validating this field like we do in some other forms (such as HCA), should we be checking for a minimum age?

Reply:
Yeah, might be worth doing.